docs/guides/quick-start/miranum-connect/worker-quickstart.md

```diff
@@ -0,0 +1,92 @@
+---
+id: worker-quickstart
+title: Create your first worker
+sidebar_label: Your first worker
+description: "This section introduces the concept of an worker and how to create a simple one."
+---
+
+# Creating Workers with Miranum Connect
+
+This guide will help you understand how to create a Miranum Worker in your Java application using Miranum Connect.
+Workers are essential components of the Miranum ecosystem that bridge the gap between your process engine and your tasks,
+enabling streamlined process automation.
+
+:::info What You Will Learn
+- How to incorporate necessary dependencies to use Miranum Message
+- How to implement and use the Message Correlation feature
+:::
+
+:::note
+This guide assumes familiarity with Java and Maven.
+:::
+
+## What is a Miranum Worker?
+
+A Miranum Worker interfaces with your process engine (like Camunda Platform 7 or 8) and fetches tasks of a specific type.
+These tasks can then trigger various actions in the context of a process instance. After a task is completed, 
+the process engine advances to the next step, maintaining a smooth flow of operations.
+
+## Add the Necessary Dependencies
+
+To start creating Workers, you'll need to add the Miranum `worker-api` dependency to your Maven project:
+
+```xml
+<dependency>
+ <groupId>io.miragon.miranum</groupId>
+ <artifactId>worker-api</artifactId>
+ <version>${miranum.worker.version}</version>
```

Comment:
⛏️ I think we could leave it this way or change it according to the previous section to `${miranum.version}`